src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/__tests__/Doubles/DownloadedPackagesStoreErrorStub.swift

```diff
@@ -76,4 +76,6 @@ class DownloadedPackagesStoreErrorStub: DownloadedPackagesStoreV2 {
 
 	var keyValueStore: Store?
 
+	var revokationList: [String] = []
```

Comment:
I don't see that .revokationList is ever written besides of the tests.
Can we write the test a different way to prevent this list?

Reply:
I thought it was also written in the real app on fetching a new app config. The idea was to prevent storing invalid packages. That is an edge case but covers a case where the CDN still provides invalid packages that have already been flagged in the app config.
I'll update the code.